custom_test.py

```
@@ -0,0 +1,128 @@
+                        idea.reasoning[:200] + "..."
+                        if len(idea.reasoning) > 200
+                        else idea.reasoning
+                    )
+                    print(f"     💭 {reasoning_preview}")
+                if idea.confidence_score:
+                    print(f"     📊 Confidence: {idea.confidence_score}")
+
+        print(f"\n🎨 Total synthesized ideas: {len(result.synthesized_ideas)}")
+        return result
+
+    except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)